packages/forms/src/model.ts

```diff
@@ -179,15 +178,13 @@ export abstract class AbstractControl {
   _hasOwnPendingAsyncValidator = false;
 
   /** @internal */
-  // TODO(issue/24571): remove '!'.
-  _pendingTouched!: boolean;
+  _pendingTouched: boolean = false;
 
   /** @internal */
   _onCollectionChange = () => {};
 
   /** @internal */
-  // TODO(issue/24571): remove '!'.
-  _updateOn!: FormHooks;
+  _updateOn: FormHooks = 'change';
```

Comment:
I'm wondering if this change is going to affect the logic in the getter here: https://github.com/angular/angular/blob/master/packages/forms/src/model.ts#L439. Currently when the `_updateOn` is undefined, we consult a parent control. With this change, the `_updateOn` will always be defined, so we would never consult a parent control. I think that we should allow the `_updateOn` property to be undefined/optional (and keep it undefined by default).

```suggestion
  _updateOn?: FormHooks;
```

It should *not* be a breaking change, since the field is internal (not a part of the public API) and the `updateOn` getter (the public API) already accounts for this case.

Reply:
It is definitely so, I changed it. Thanks a lot!